webapp/first_snap/content/ros2/test.yaml

```diff
@@ -0,0 +1,16 @@
+macos:
+  - action: Create a Linux virtual machine for testing snaps
+    command: multipass launch -n testvm
+  - action: Return to the directory containing the .snap file and copy it into the virtual machine
+    command: 'multipass copy-files ros2-talker-listener*.snap testvm:'
+  - action: Connect to the virtual machine
+    command: multipass shell testvm
+  - action: Install the snap inside the virtual machine
+    command: sudo snap install --dangerous ros2-talker-listener*.snap
```

Comment:
Missing a `--devmode` here. Pro tip: `sudo snap install --devmode` actually implies `--dangerous` (at least currently), but not the other way around.

Reply:
@evandandrea Same question as in the ROS guide - can we make a site-wide change for the test template?